h2d/Text.hx

```diff
@@ -371,10 +382,11 @@ class Text extends Drawable {
 				}
 				else wLastSep = size;
 			}
-			else if( (x + esize + letterSpacing) - wLastSep > maxWidth ) {
+			else if( (x + esize + letterSpacing) - wLastSep > (maxWidth-esize) && wordWrap ) {

```

Comment:
Probably should reverse the order of checks, and put `wordWrap` first, as first check becomes redundant when it's off.